.kitchen.yml

```diff
@@ -58,6 +58,27 @@ suites:
             major_version: 5
             plugins:
               lang-python: lang-python
+            jvm_opts:
+              - -Xms2g
```

Comment:
You only need one of these to test the functionality.

In addition, do you want to write a test case for this? As usual, not required.